Jenkinsfile.deploy

```diff
@@ -28,104 +41,86 @@ pipeline {
     }
 
     environment {
+        JAVA_HOME = "${GRAALVM_HOME}"
```

Comment:
not sure if use graalvm as JAVA home is a good idea.
probably define JAVA_home with the openjdk and GRAALVM_HOME would be better,

Reply:
we need to build native example locally with Maven so Graalvm needs to be default for the tests (it is only used there)...